apps/multitimer/app.js

```diff
@@ -631,6 +655,11 @@ function editAlarm(idx, a) {
     },
     "Hours": {
       value: t.hrs, min: 0, max: 23, wrap: true,
+      format: v => {
+        // Show 12h or 24h based on locale settings
+        const mer = require("locale").meridian(new Date(1999,1,1,v,0,0));
+        return (!mer) ? v : (v%12||12) + mer;
```

Comment:
This might be more easily read as:

```suggestion
        return mer ? (v%12||12) + mer : v;
```